ee/codegen/src/context/workflow-context/workflow-context.ts

```diff
@@ -98,6 +106,7 @@ export class WorkflowContext {
     this.parentNode = parentNode;
 
     this.sdkModulePathNames = generateSdkModulePaths(workflowsSdkModulePath);
+    this.workflowRawEdges = workflowRawEdges;
```

Comment:
nit 2/5: If we're adding edges to `WorkflowContext`, it'd be great to remove them from `Workflow.Args` since it's now kinda duplicative. Note that if we do this, there might be wonky stuff cuz I think I filter out edges in some places.

Reply:
If you do decide to take this one, I suggest merging this as is and doing as a follow-up.